package-res/resources/web/pentaho/visual/base/View.js

```diff
@@ -64,75 +42,176 @@ define([
+       * @readOnly
+       * @private
+       */
       this.__changeDidHandle = model.on("did:change", this.__onChangeDidOuter.bind(this));
     },
 
     //region Properties
     /**
-     * Gets the view's DOM node.
+     * Gets the container DOM element where the view is rendered.
+     *
+     * The container element is the viewport through which the view's rendered content is revealed.
+     * and is provided at construction time, empty, for its exclusive use, by the _container application_.
+     *
+     * Its _content_ is owned by the view,
+     * but its attributes (including style) are owned by the container application and
+     * must **not** be changed by the view.
+     *
+     * Its size is controlled by the container application and does not need to be the same as
+     * that implied by the visual model's
+     * [width]{@link pentaho.visual.base.Model#width} and
+     * [height]{@link pentaho.visual.base.Model#height} properties.
      *
-     * @type {?(Node|Text|HTMLElement)}
+     * It is the responsibility of the container application to clean up the container element,
+     * if needed, after the view is disposed.
+     *
+     * @type {!DOMElement}
+     * @readOnly
      */
     get domContainer() {
       return this.__domContainer;
     },
 
+    /**
+     * Gets the visualization model.
+     *
+     * @type {!pentaho.visual.base.Model}
+     * @readOnly
+     * @see pentaho.visual.base.View#context
+     */
     get model() {
       return this.__model;
     },
 
     /**
-     * Gets the context of the view's model.
+     * Gets the type context of the view's model.
      *
      * This getter is syntax sugar for `this.model.type.context`.
      *
-     * @type {pentaho.type.Context}
+     * @type {!pentaho.type.Context}
+     * @readOnly
+     * @see pentaho.visual.base.View#model
      */
     get context() {
       return this.model.type.context;
     },
 
     /**
-     * Gets the value that indicates if an update is in progress.
+     * Gets a value that indicates if an update is in progress.
      *
-     * @type {!boolean}
+     * @type {boolean}
+     * @readOnly
+     * @see pentaho.visual.base.View#update
      */
     get isUpdating() {
       return !!this.__updatingPromise;
     },
 
     /**
-     * Orchestrates the rendering of the visualization and is meant to be invoked by the container.
+     * Gets or sets a value that enables or disables automatic updates of the view.
+     *
+     * When `true`, the view is automatically updated whenever a relevant change in the model occurs.
+     * When `false`, the view must be manually updated by calling [update]{@link pentaho.visual.base.View#update}.
+     *
+     * Setting this property to `true` does **not** cause the view to update in response,
+     * if it is currently [dirty]{@link pentaho.visual.base.View#isDirty}.
+     * This property only affects the view's behaviour in response to subsequent model changes.
```

Comment:
This property only affects the view's behavior in response to subsequent model changes.